auto3dseg/notebooks/msd_datalist_generator.ipynb

```diff
@@ -0,0 +1,206 @@

```

Comment:
Would be nice to cut down the length of output as mentioned in the previous comments.